src/module.c

```diff
@@ -6461,6 +6461,38 @@ const char *RM_CallReplyProto(RedisModuleCallReply *reply, size_t *len) {
     return callReplyGetProto(reply, len);
 }
 
+/* Redis only exposes "side effects" of command execution at the end of an execution unit.
+ *
+ * Side effects include propagating replication/AOF operations and executing the commands of clients blocked on keys
+ * modified by commands
+ *
+ * In general, for modules, execution unit start and end with the taking/release of the GIL.  For module code executed
+ * on the Redis main thread, this is done implicitly by Redis itself, while for module code executed on a separate
+ * thread, this is done explicitly by the module itself.
+ *
+ * This function enables modules to flush the execution unit, thereby having side effects become visible during
+ * execution.
+ *
+ * Examples of usage might be that one is running multiple commands, one after another, with RM_Call and wants
+ * processCommand() like semantics, where unblocked clients will be executed before the next RM_Call is made.
+ *
+ * Note: while one Modules that calls another module via RM_Call won't nest if they are operating on a separate thread,
```

Comment:
```suggestion
 * Note: while one module that calls another module via RM_Call won't nest if they are operating on a separate thread,
```

Reply:
i'm actually not sure if all that detail is necessary in the API documentation.
e.g. `it doesn't make sense for the further nested module`...